action.yaml

```diff
@@ -23,7 +23,7 @@ inputs:
   overwrite:
     description: "Overwrite target container image with slimmed version (only if target is not ID)"
     required: false
-    default: false
+    default: "false"
```

Comment:
Why is this wrapped in double quotes?

Reply:
![image](https://github.com/user-attachments/assets/dd1664b8-429e-4c70-8483-23d9beaed3b6)

A string representing the default value. The default value is used when an input parameter isn't specified in a workflow file.

vscode code get me  error info.

Reply:
Confirmed.

https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault